crates/rbuilder/src/live_builder/block_output/bidding/parallel_sealer_bid_maker.rs

```diff
@@ -0,0 +1,161 @@
+use reth_primitives::format_ether;
+use std::sync::{Arc, Mutex};
+use tokio::sync::Notify;
+use tokio_util::sync::CancellationToken;
+use tracing::{error, info};
+
+use crate::live_builder::block_output::relay_submit::BlockBuildingSink;
+
+use super::interfaces::{Bid, BidMaker};
+
+/// BidMaker with a background task sealing multiple parallel bids concurrently.
+/// If several bids arrive while we hit the max of concurrent sealings we keep only the last one since we assume new is better.
+#[derive(Debug)]
+pub struct ParallelSealerBidMaker {
+    pending_bid: Arc<PendingBid>,
+}
+
+impl BidMaker for ParallelSealerBidMaker {
+    fn send_bid(&self, bid: Bid) {
+        let payout_tx_value = bid.payout_tx_value();
+        let block = bid.block();
+        info!(
+            bid = if let Some(bid) = payout_tx_value {
+                format_ether(bid)
+            } else {
+                "None".to_string()
+            },
+            can_add_payout_tx = block.can_add_payout_tx(),
+            tbv = format_ether(block.true_block_value().unwrap_or_default()),
+            "DX send_bid"
+        );
+        self.pending_bid.update(Bid::new(block, payout_tx_value));
+        //        self.pending_bid.update(bid);
```

Comment:
```suggestion
```

Reply:
Ouch that info! was supposed to be removed!! Sorry! I'll fix that.